src/kili/queries/asset/helpers.py

```diff
@@ -24,7 +27,16 @@ def get_post_assets_call_process(
     Otherwise return assets without any post-processing
     """
     if download_media:
-        projects = kili.projects(project_id, fields=["inputType"])
+        where = ProjectWhere(
```

Comment:
at some point we should add a core `Project` object from which we could get a field. It would live between the services and the queries